setup.py

```diff
@@ -40,6 +40,7 @@ def is_new_osx():
     "spacy.lexeme",
     "spacy.vocab",
     "spacy.attrs",
+    "spacy.kb",
```

Comment:
@honnibal Do you think `kb` is okay for a module name here? Alternative could be `spacy.knowledge`, which I like as well. (But might be one of those words that's annoying to spell / easy to misspell?)

Reply:
I dunno, `knowledge` is a bit vague. It sounds like it holds *all* the knowledge.

On the topic of naming, I wonder whether `KnowledgeBase` is actually a bit problematic? It sounds like the base class of the `Knowledge` hierarchy, which is quite emphatically untrue. We might disprefer `spacy.knowledge_base.pyx` as a module name for similar reasons.

Maybe we want `spacy.entities`, to contrast with `spacy.tokens`?

Reply:
Yeah, I know what you mean. I don't really have a better idea either, though.

> Maybe we want `spacy.entities`, to contrast with `spacy.tokens`?

Hmm, not sure – I think that'd also be misleading, because people would expect the entity recognizer and entity ruler to live there (and IMO those should stay in `spacy.pipeline`, because they're pipeline components).

Reply:
In EL literature, `knowledge base` is by far the most often used term. Other options I can think of are `thesaurus`/`ontology` or `concepts`. Let me know what you guys decide :-)

Reply:
Let's stick with `KnowledgeBase`, which I think makes the module `kb`. We can always change it before we ship, so let's not get hung up on it for now.